linkerd/core/src/main/scala/io/buoyant/linkerd/Server.scala

```diff
@@ -102,7 +111,8 @@ class ServerConfig { config =>
     label.getOrElse(routerLabel),
     port.map(_.port).getOrElse(pi.defaultServerPort),
     pi,
-    serverParams
+    serverParams,
+    announce.getOrElse(false)
```

Comment:
I think we should default to true.  If I have an announcers block, I want things to be announced.